src/util/sentry.ts

```diff
@@ -16,26 +16,46 @@ const isInformationalError = (exception: unknown): boolean => {
 };
 
 const sentryIgnoreErrors = [
+  // Network problems
+  "Failed to fetch",
```

Comment:
Kan ikke denne bli litt "catch-all"-y?
Ser for meg at vi kunne laget en feil som er "Failed to fetch xyz because of this and that" som ikke ville blitt med siden denne fanger alle feil som inneholder denne teksten?